src/module.ts

```diff
@@ -55,14 +55,9 @@ export default defineNuxtModule<ModuleOptions>({
     logger.info('Setting up auth...')
 
     // 2. Set up runtime configuration
-    let usedOrigin
-    const isOriginSet = typeof moduleOptions.origin === 'undefined'
```

Comment:
This was definitely the opposite from what it should have been 😆

Reply:
Ooof